ExampleApp/package.json

```diff
@@ -3,20 +3,24 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "start": "node node_modules/react-native/local-cli/cli.js start",
-    "test": "jest"
+    "start": "react-native start",
+    "test": "jest",
+    "lint": "eslint ."
   },
   "dependencies": {
-    "@wowmaking/react-native-iron-source": "^2.8.5",
-    "react": "16.6.3",
-    "react-native": "0.58.4"
+    "@wowmaking/react-native-iron-source": "^3.0.0",
```

Comment:
Let's use `extraNodeModules` in metro config to use current version from file system

Reply:
Added in f1513698d9680a336db49bdc2ade360f4263ae55

Reply:
`"@wowmaking/react-native-iron-source": "^3.1.0"` is it still required?

Reply:
Nope.
See c9f8900711a2d504c0e9915824e18e826b5a661e where I deleted it.